resources/assets/sass/app.scss

```diff
@@ -5147,11 +5147,6 @@ float: left;
   margin-left:10px;
 }
 
-.modal-open .select2-container{
```

Comment:
Why is this removed? Is it related to your change somehow?  If not, please revert this change

Reply:
@fyhring ^

Reply:
Sorry. I made a mistake. I meant to make two diffent PR's. I'll try to revert.

Reply:
No worries.  Always do work in branches.  PRs aren't a "snapshot" of the changes at that point, they're a complete list of commits that differ between the source and destination branches and are therefore updated automatically if new commits are pushed.